### DISCUSSION TITLE
Tailwind CSS v3.0: Just-in-Time, all the time

### DISCUSSION THREAD

**Discussion by adamwathan** [tailwindlabs/tailwindcss#6317] [Announcements]
Tailwind CSS v3.0 is here — bringing incredible performance gains, huge workflow improvements, and a seriously ridiculous number of new features.

https://user-images.githubusercontent.com/4323180/145496268-8ab93a1f-bcda-435e-8ed0-2ec46a05ba94.mp4

Read all about it in [the Tailwind CSS v3.0 announcement post](https://tailwindcss.com/blog/tailwindcss-v3) and let us know what you think! 🥳

**Comment by NovusOpus** [comment]
Congrats for the Launch 🎉 , And Thanks for such an Amazing frameworks that makes many Dev's work easier and joyful.

**Comment by BleedingEffigy** [comment]
Sheeeesh! I am honestly shocked at all those new colors. And from the scroll snapping to the shadows and everything else, I have so many modifications I want to make to my old websites. Oh and thank goodness for auto JIT. Congrats!

**Comment by Sean-Sophearom** [comment]
Hi, I'm sorry if this isn't the right place to post this but I can't find anywhere better. I was just reading the docs and saw that the "marker:" modifier, which styles the counters or bullets in list, is kind of lost in the "Hover, Focus and Other States" section and hard to find. Perhaps, there should be a section for it somewhere near the "List Style Type" and "List Style Position" section. Loving the new version by the way. Thanks for all the hard work.

**Reply by adamwathan** [reply]
Great point, I will make a note to mention this feature in the list utilities documentation 👍🏻 Thank you!

**Comment by larrasu** [comment]
Amazing! Words are not enough. Really superb framework for something free, but feels like premium. Cheers 🥂

**Comment by surjithctly** [comment]
Team, It seems the typography plugin is still in alpha. That needs to be released now as well right?

**Comment by zaarheed** [comment]
Congrats on the release! The [Upgrade Guide](https://tailwindcss.com/docs/upgrade-guide) from v2 to v3 doesn't make any explicit mention to what to do with `@tailwindcss/jit`. Later, the page mentions removing `mode: jit` from the tailwind config file. But I think there might be value in explicitly saying developers can uninstall `@tailwindcss/jit`?

Any thoughts?

**Reply by adamwathan** [reply]
Hey! Yep definitely uninstall that, that has been obsolete since v2.1 back in the spring.
